src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/TestGenieToolWindow.kt

```diff
@@ -56,66 +74,32 @@ class TestGenieToolWindow {
         .setFormLeftIndent(30)
         .addVerticalGap(25)
         .addComponent(panelTitle)
+        .addComponent(JXTitledSeparator("Search budget"), 35)
+        .addLabeledComponent(customLabel("Search budget type",
+            "What condition should be checked to end the search."), stoppingCondition, 25, false)
+        .addTooltip(default(StoppingCondition.MAXTIME.toString()))
         .addLabeledComponent(customLabel("Search budget", "Maximum search duration."), searchBudget, 25, false)
-        .addTooltip(default("60 seconds"))
```

Comment:
General remark: This section will need to be double checked when merging this branch's changes with #1

Reply:
I have even triple-checked that